app/templating/template_renderer.py

```diff
@@ -15,10 +15,12 @@ def __init__(self):
 
         self.environment.filters['format_date'] = format_date
         self.environment.filters['format_str_as_date'] = format_str_as_date
-        self.environment.filters['format_currency'] = format_currency
         self.environment.filters['format_household_name'] = format_household_member_name
         self.environment.filters['format_household_summary'] = format_household_summary
         self.environment.globals['format_conditional_date'] = format_conditional_date
+        self.environment.filters['format_currency'] = format_currency
```

Comment:
How about we switch to using [Jinja filter decorators](http://flask.pocoo.org/docs/0.12/templating/#registering-filters) in app.jinja_filters?